Source/Resources/Content/MSGraph/DeviceManagement/DeviceEnrollmentConfigurations/windows10EnrollmentCompletionPageConfiguration - All users and all devices.json

```diff
@@ -12,7 +12,7 @@
   ],
   "blockDeviceSetupRetryByUser": true,
   "customErrorMessage": "",
-  "description": "This is the default enrollment status screen configuration applied with the lowest priority to all users and all devices regardless of group membership.",
+  "description": "What is it: The default Intune enrollment status page configuration applied with the lowest priority to all users and all devices regardless of group membership. \n Simeon Notes: The enrollment status page is a visual queue to users of the device enrollment progress. Simeon's baseline allows the user to select the 'Continue Anyway' option as soon as it is available and times out after 40 minutes. \n Learn More: https://docs.microsoft.com/en-us/mem/intune/enrollment/windows-enrollment-status",
```

Comment:
**What does this do?**
Configures the page that is displayed to users while a device enrollment is in progress. The baseline allows the user to select the 'Continue Anyway' option as soon as it is available and times out after 40 minutes.

**Why should you use this?**
If you want to allow users to click 'Continue Anyway' when a single configuration in the enrollment process fails and you don't want users to be stuck on the enrollment screen indefinitely.

**What is the end-user impact?**
Users will be able to click 'Continue Anyway' when a single configuration in the enrollment process fails.

**Learn more**
[Set up the Enrollment Status Page](https://docs.microsoft.com/en-us/mem/intune/enrollment/windows-enrollment-status)

Reply:
@david-simeon  updated a little take another look.